src/Uno.UWP/UI/Notifications/ToastNotifier.Android.cs

```diff
@@ -228,7 +228,7 @@ private Android.App.PendingIntent CreateToastIntent(string packageName, XmlDocum
 				var childLaunch = childToast[0]?.Attributes?.GetNamedItem("launch");
 				if (childLaunch != null)
 				{

```

Comment:
Could this do instead?

```suggestion
				toastArgument = childLaunch?.Value;
```

Reply:
No, because compiler says `Error	CS8600	Converting null literal or possible null value to non-nullable type`.  (toastArgument  is string). And in intentAction.PutExtra(), I should put empty string and not null.

This line was `toastArgument = childLaunch.Value`, but then compiler says (CI build, not Intellisense) `possible null refernce`.

Reply:
But is is ridiculous - it is inside `if` checking `childLaunch` for being not null. So, it is compilers error, and `!` seems better ("hey, compiler, here childLaunch is really not null!") than `Value ?? String.Empty` (such construction suggests that it can be null, but it cannot).